app/notifications/notifications.component.html

```diff
@@ -2,11 +2,25 @@
   <h1>{{ 'my notifications' | translate }}</h1>
 </md-card>
 <md-card class="default-card">
+  <div class="filter-options" *ngIf="notifications.length != 0">
+    <span>{{ 'FILTER BY' | translate }}</span>
+    <md-select [(ngModel)]="selectedStatus" placeholder="{{ 'Status' | translate }}">
+      <md-option *ngFor="let statusOption of notificationStatusOptions" [value]="statusOption">
+        {{ statusOption | translate }}
+      </md-option>
+    </md-select>
+    <md-select class="select-layout" [(ngModel)]="selectedNotificationType" placeholder="{{ 'Notification types' | translate }}">
+      <md-option *ngFor="let notificationType of notificationTypes" [value]="notificationType">
+        {{ notificationType | translate }}
+      </md-option>
+    </md-select>
+  </div>
```

Comment:
Can you put the status select to the right in top of the "mark as read" button?
And the Notification Type to the right, top of the Notification?

Maybe if you do so, you can remove the "Filter by" span because it is clear what it means? @patrickrobrecht what do you think?